pkgs/applications/science/math/mxnet/default.nix

```diff
@@ -13,25 +13,26 @@ assert cudnnSupport -> cudaSupport;
 
 stdenv.mkDerivation rec {
   pname = "mxnet";
-  version = "1.8.0";
+  version = "1.9.1";
 
   src = fetchurl {
     name = "apache-mxnet-src-${version}-incubating.tar.gz";
     url = "https://dlcdn.apache.org/incubator/mxnet/${version}/apache-mxnet-src-${version}-incubating.tar.gz";
-    hash = "sha256-la/5hYlaukCcCNVRRRCuOLiEkM+2KBqzpf8PWCbI21Q=";
+    hash = "sha256-EephMoF02MKblvNBl34D3rC/Sww3rOZY+T442euMkyI=";
   };
 
   patches = [
-    # Fix build error https://github.com/apache/incubator-mxnet/issues/19405
+    # Remove the following two patches when updating mxnet to 2.0.
     (fetchpatch {
-      name = "mxnet-fix-gcc-linker-error-1.patch";
-      url = "https://github.com/apache/incubator-mxnet/commit/78e31d66d19e385ca4ef73245ce79a47e375d8d1.diff";
-      sha256 = "sha256-UfmGhh4RbvrEOXe6IJxHm1Aqpy1gS6gHxrX5KQBXjv4=";
+      name = "1-auto-disable-sse-for-non-x86.patch";
+      url = "https://github.com/apache/incubator-mxnet/commit/55e69871d4cadec51a8bbb6700131065388cb0b9.patch";
```

Comment:
Does this commit have a variant that exists on an upstream branch? If not does it belong to an unmerged PR you could link to?

Reply:
Sorry, I mean the other patch :point_down:

Reply:
Found it and updated the PR. Thanks for pointing it out.